lib/source_file.h

```diff
@@ -306,6 +323,7 @@ class InputSources final {
 
     cstring toDebugString() const;
     void addComment(SourceInfo srcInfo, bool singleLine, cstring body);
+    const std::vector<Comment *> &getAllComments() const;
```

Comment:
So does this returns all comments in the file? If so, I don't think it logically belongs to `SourceInfo`. And if we want it there, the name should be clearer. From the name, I would expect all comments attached to the source info (i.e. the once before and after).

Reply:
I would also add a comment describing the method and its purpose.